slim/Dockerfile.armhf

```diff
@@ -0,0 +1,52 @@
+FROM arm32v6/alpine
```

Comment:
I'd prefer to avoid the exact copy of amd64 Dockerfile. Do you know that in case of such copypaste the copies will always diverge, and various changes will accidentally go to only one copy?